ocaml/idl/ocaml_backend/gen_server.ml

```diff
@@ -211,7 +211,7 @@ let operation (obj : obj) (x : message) =
   let rbac_check_begin =
     if has_session_arg then
       [
-        "let arg_names = "
```

Comment:
Previously an error in this code resulted in the RBAC checks failing due to 'wrong length'. Now it will fail here, I-m not sure how the exception will manifest. It's worth mentioning that all `List.combine` usages are done inside a pattern match that enforces the number of elements in the list `__params`.